test/spiretest/assertions.go

```diff
@@ -46,7 +46,7 @@ func AssertGRPCStatus(tb testing.TB, err error, code codes.Code, message string)
 	st := status.Convert(err)
 
 	ok := true
-	if !assert.Equal(tb, code, st.Code(), "GRPC status code does not match") {
+	if !assert.Equal(tb, code.String(), st.Code().String(), "GRPC status code does not match") {
```

Comment:
This isn't really related but makes debugging test failures much easier since you see the string representation of the code instead of the integer value.